test/networkd-test.py

```diff
@@ -280,15 +281,15 @@ class ClientTestBase(NetworkdTestingUtilities):
     """Provide common methods for testing networkd against servers."""
 
     @classmethod
-    def setUpClass(klass):
-        klass.orig_log_level = subprocess.check_output(
+    def setUpClass(cls):
+        cls.orig_log_level = subprocess.check_output(
```

Comment:
`cls` may be a tiny bit more idiomatic than `klass`, but I don't see the point of this change and similar ones below.

Reply:
It's to follow python convention of having the class argument being called `cls`. Which makes it easier for new users and tools like pylint to know what it does